libs/webb-ui-components/src/utils/formatTokenAmount.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Format token amount to 3 decimal places and add `<` if amount is less than 0.001
+ * @param amount The amount to format
+ * @returns The formatted amount
+ */
+export const formatTokenAmount = (amount: string) => {
```

Comment:
Can you make the `decimal places` as the argument of this function and set it equal to `3` by default? Also, it seems the function document is not matched with the function implementation `less than 0.001` but in the function `< 0.01`.